cmd/api/src/migrations/manifest.go

```diff
@@ -47,6 +47,37 @@ func RequiresMigration(ctx context.Context, db graph.Database) (bool, error) {
 	}
 }
 
+// Version_730_Migration removes the leftover 'adminrightscount' property from User nodes
+func Version_730_Migration(ctx context.Context, db graph.Database) error {
+	const adminRightsCount = "adminrightscount"
+
+	defer measure.LogAndMeasure(slog.LevelInfo, "Migration to remove admin_rights_count property from user nodes")
+
+	return db.BatchOperation(ctx, func(batch graph.Batch) error {
```

Comment:
I would just replace this with a db.ReadTransaction, or just do a WriteTransaction here. The batch is effectively unused.